src/docs/sdk/performance/modules/llm-monitoring.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 'LLM Monitoring'
+---
+
+Sentry auto-generates LLM Monitoring data for common providers in Python, but you may need to manually annotate spans for other frameworks.
+
+## Span conventions
+
+### Span Operations
+
+| Span OP         | Description                                                                          |
+|:----------------|:-------------------------------------------------------------------------------------|
+| `ai.pipeline.*` | The top-level span which corresponds to one or more AI operations & helper functions |
+| `ai.run.*`      | A unit of work - a tool call, LLM execution, or helper method.                       |
+
+### Span Data
+
+| Attribute                   | Type    | Description                                           | Examples                                 |
+|-----------------------------|---------|-------------------------------------------------------|------------------------------------------|
+| `ai.input_messages`         | string  | The input messages sent to the model                  | `[{"role": "user", "message": "hello"}]` |
+| `ai.completion_tоkens.used` | int     | The number of tokens used to respond to the message   | `10`                                     |
+| `ai.prompt_tоkens.used`     | int     | The number of tokens used to process just the prompt  | `20`                                     |
+| `ai.total_tоkens.used`      | int     | The total number of tokens used to process the prompt | `30`                                     |
+| `ai.model_id`               | list    | The vendor-specific ID of the model used              | `"gpt-4"`                                |
+| `ai.streaming`              | boolean | Whether the request was streamed back                 | `true`                                   |
+| `ai.responses`              | list    | The response messages sent back by the AI model       | `["hello", "world"]`                     |
+| `ai.pipeline.name`          | string  | The description of the parent ai.pipeline span        | `My AI pipeline`                         |
```

Comment:
Are these all optional or are some required?

Reply:
everything but `streaming` is required